src/blog/2024/10/dashboard-new-group-type-app-icon-and-charts.md

```diff
@@ -0,0 +1,60 @@
+---
+title: New Chart Type, Customizable Icon and Groups as Dialogs Now Available in FlowFuse Dashboard
+subtitle: Our latest update for FlowFuse Dashboard introduces a new group type, Dialog, a new chart variation, Histogram and customization support for the application icon.
+description: Our latest update for FlowFuse Dashboard introduces a new group type, Dialog, a new chart variation, Histogram and customization support for the application icon.
+date: 2024-10-11
+authors: ["gayan-sandamal"]
+image: /blog/2024/08/images/dashboard-2-1150-release.png
+tags:
+   - posts
+   - news
+   - flowfuse
+   - dashboard
+---
+
+At FlowFuse, we strive to improve your dashboard experience by introducing features that enhance data visualization and customization. This update brings new ways to tailor your dashboards with key improvements, including a new chart type, a customizable app icon, and a dialog feature for groups.
+
+<!--more-->
+
+## New Chart Type: Histogram
+
+Histograms are an essential tool for data analysis, offering a clear way to visualize distributions. The latest FlowFuse update introduces a fully customizable histogram chart type, allowing you to present frequency distributions for data values. Whether analyzing performance metrics or user activity, histograms give you a clear view of how data points are distributed across predefined ranges. You can now easily group data and control the range, providing deeper insights at a glance.
+
+![Screenshot showing the new Histogram chart type in the FlowFuse Dashboard](./images/chart-histogram.png){data-zoomable}
+_Screenshot showing the new Histogram chart type in the FlowFuse Dashboard_
+
+With this feature, visualizing data distribution is easier than ever. Histograms are ideal for applications like monitoring error rates, analyzing sensor data, or displaying survey results.
+
+## Customizable App Icon
+
+Branding is an essential part of any user experience and with this new feature, you can customize your dashboard's app icon. The FlowFuse Dashboard now allows users to provide their own application icon, which appears in the browser tab and when the dashboard is installed as a Progressive Web App (PWA). This customization helps reinforce your brand, whether you’re developing IoT solutions, monitoring systems or creating dashboards for end-users.
+
+![Screenshot showing the customizable app icon in browser and as a PWA](./images/app-icon-installation.png){data-zoomable style="max-width: 400px; margin: auto;"}
```

Comment:
I'd go with a different screenshot here, or at least one that doesn't show `localhost`, looks a little unprofessional

Reply:
Took and replaced the screenshot with the interactive dashboard's demo instance with the PWA install prompt